libpod/image/pull.go

```diff
@@ -176,6 +176,24 @@ func (ir *Runtime) pullGoalFromImageReference(ctx context.Context, srcRef types.
 		}, nil
 
 	case OCIArchive:
+		// imgName such as "oci-archive:/tmp/FOO.tar:localhost/foo:latest"
+		// or "oci-archive:/tmp/FOO.tar:domain.com/foo:tag1", "oci-archive:/tmp/FOO.tar"
+		imgNameSli := strings.SplitN(imgName, ":", 3)
+		if len(imgNameSli) == 3 {
+			// Fixes: https://github.com/containers/podman/issues/7337
```

Comment:
- How does this relate to #7337 _at all_, when the previous version did not use the path component of `srcRef` for `dest` either AFAICS?
- Why is fixing the single-image case enough? Can’t multi-image archives use upper-case names as well?

Reply:
Still no answer to this.